runner/src/simulate.rs

```diff
@@ -628,17 +654,23 @@ impl Metrics {
         })?;
         Ok(Self { inner })
     }
-    fn record(&mut self, metrics: GooseMetrics) {
+    fn record(&mut self, metrics: GooseMetrics, min_peer_rps: Option<f64>) {
         let mut gm = self
             .inner
             .lock()
             .expect("should be able to acquire metrics lock for mutation");
         gm.goose_metrics = Some(metrics);
+        gm.min_peer_rps = min_peer_rps;
     }
 }
 
 impl MetricsInner {
     fn observe(&mut self, cx: &Context) {
+        // TODO add simulation specific attributes
```

Comment:
was going to comment on this, but never mind for now. we should be able to keep going like this but it'd be nice to have some label to make sure we're comparing the same thing over time